sdk/ml/azure-ai-ml/azure/ai/ml/constants/_job/job.py

```diff
@@ -107,7 +107,7 @@ class RestNames:
 
     REST_TO_ENTITY = {v: k for k, v in ENTITY_TO_REST.items()}
 
-    NAMES_ALLOWED_FOR_PUBLIC = [EntityNames.JUPYTER_LAB, EntityNames.SSH, EntityNames.TENSOR_BOARD, EntityNames.VS_CODE]
+    NAMES_ALLOWED_FOR_PUBLIC = [EntityNames.CUSTOM, EntityNames.TRACKING, EntityNames.STUDIO]
```

Comment:
IIRC, we need users to pass in EntityNames.JUPYTER_LAB, EntityNames.SSH, EntityNames.TENSOR_BOARD, EntityNames.VS_CODE. Should we include them as well?